Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -370,52 +368,42 @@ extension TestListViewController: UITableViewDelegate, UITableViewDataSource {
             currentCycleIndex = indexPath.row - selectedPlanIndex - testRunList.testRunList.count - 1
         }
         
-        let cell = UITableViewCell(style: UITableViewCellStyle.default, reuseIdentifier: "TestCycleCell")
-        
-        cell.textLabel?.text = self.testCycleList.testCycleList[currentCycleIndex].name
-        cell.textLabel?.textAlignment = .left
-        cell.textLabel?.font = UIFont(name: "Helvetica Neue", size: 20.0)
+        let cell = TestListTableViewCell(style: .default, reuseIdentifier: "TestCycleCell")
+        cell.customInit(tableWidth: testList.frame.width, cellType: .testCycle)
+        cell.nameLabel.text = self.testCycleList.testCycleList[currentCycleIndex].name
         
         //If cycle's cell is selected change the background color
-        if selectedCycleTableViewIndex != indexPath.row {
-            cell.backgroundColor = UIColor(colorLiteralRed: 0xE5/0xFF, green: 0xE5/0xFF, blue: 0xE5/0xFF, alpha: 1)
-        } else {
-            cell.backgroundColor = UIColor(colorLiteralRed: 0xFF/0xFF, green: 0xFD/0xFF, blue: 0xCF/0xFF, alpha: 1)
-            
+        if selectedCycleTableViewIndex == indexPath.row {
+            cell.selectCell()
         }
-        cell.indentationLevel = 1
-        cell.indentationWidth = 15.0
+
         if(self.testCycleList.testCycleList[0].name == "No Cycles Found")
         {
```

Comment:
I know this wasn't part of your changes, but can you move this bracket to the line above it?